code.gs

```diff
@@ -1475,27 +1475,27 @@ var i18n = {
   },
   'id': {
     'Age': 'Usia',
-    'Years': 'Tahun-tahun',
-    'Events': 'Peristiwa-peristiwa',
+    'Years': 'Tahun',
+    'Events': 'Acara',
     'Birthdays today': 'Ulang tahun hari ini',
     'Birthdays tomorrow': 'Ulang tahun besok',
     'Birthdays in {0} days': 'Ulang tahun dalam {0} hari mendatang',
     'Anniversaries today': 'Hari jadi hari ini',
     'Anniversaries tomorrow': 'Hari jadi besok',
     'Anniversaries in {0} days': 'Hari jadi dalam {0} hari mendatang',
-    'Custom events today': 'Peristiwa khusus hari ini',
-    'Custom events tomorrow': 'Peristiwa khusus besok',
-    'Custom events in {0} days': 'Peristiwa khusus dalam {0} hari mendatang',
-    'Hey! Don\'t forget these events': 'Hai! Jangan lupa peristiwa-peristiwa berikut',
+    'Custom events today': 'Acara khusus hari ini',
+    'Custom events tomorrow': 'Acara khusus besok',
+    'Custom events in {0} days': 'Acara khusus dalam {0} hari mendatang',
+    'Hey! Don\'t forget these events': 'Hei! Jangan lupa acara ini',
     'version': 'versi',
     'dd-MM-yyyy': 'dd-MM-yyyy',
     'Mobile phone': 'Telp. Selular',
     'Work phone': 'Telp. Kantor',
     'Home phone': 'Telp. Rumah',
     'Main phone': 'Telp. Utama',
     'Other phone': 'Telp. Lain',
-    'Home fax': 'Fax Rumah',
-    'Work fax': 'Fax Kantor',
+    'Home fax': 'Faks. Rumah',
```

Comment:
Ok, from what I can see and understand the edits look OK.  
May I ask what's the reason for the point here? Is "Faks" an abbreviation for the word which translates "Fax"?

Reply:
Yes, "Faks." is an abbreviation of word "Faksimile" in Indonesian